sysdig/resource_sysdig_monitor_notification_channel_custom_webhook.go

```diff
@@ -0,0 +1,187 @@
+package sysdig
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	v2 "github.com/draios/terraform-provider-sysdig/sysdig/internal/client/v2"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceSysdigMonitorNotificationChannelCustomWebhook() *schema.Resource {
+	timeout := 5 * time.Minute
+
+	return &schema.Resource{
+		CreateContext: resourceSysdigMonitorNotificationChannelCustomWebhookCreate,
+		UpdateContext: resourceSysdigMonitorNotificationChannelCustomWebhookUpdate,
+		ReadContext:   resourceSysdigMonitorNotificationChannelCustomWebhookRead,
+		DeleteContext: resourceSysdigMonitorNotificationChannelCustomWebhookDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(timeout),
+			Update: schema.DefaultTimeout(timeout),
+			Read:   schema.DefaultTimeout(timeout),
+			Delete: schema.DefaultTimeout(timeout),
+		},
+
+		Schema: createMonitorNotificationChannelSchema(map[string]*schema.Schema{
+			"url": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"http_method": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringInSlice([]string{"POST", "PUT", "PATCH", "DELETE"}, false),
```

Comment:
Why we have this validation? Is this validation done on API as well? If yes I would not duplicate code. We do not want to maintain feature parity whenever new thing is added.